controller/src/main/java/io/pravega/controller/metrics/StreamMetrics.java

```diff
@@ -92,6 +96,11 @@ public final class StreamMetrics extends AbstractControllerMetrics {
 
     private final OpStatsLogger createScopeLatency;
     private final OpStatsLogger deleteScopeLatency;
+    // Controller EventProcessor events latency
```

Comment:
did you mean event's?

Reply:
its like All controllerProcessor events like update, delete seal etc are mentioned below this comment.